spec/presenters/budget_presenter_spec.rb

```diff
@@ -7,6 +7,11 @@
     it "returns the name of the budget type" do
       expect(described_class.new(budget).budget_type).to eq("Direct")
     end
+
+    it "returns nothing the the budget type is not present" do
```

Comment:
tiny typo here I think. 

Probably you wanted "returns nothing *when* the budget type is not present"?

Reply:
Cool - good catch 👍